lib/lookup.json

```diff
@@ -361,7 +364,8 @@
   "path-to-regexp": {
     "prefix": "v",
     "maintainers": "blakeembrey",
-    "skip": ["aix"]
+    "skip": ["aix", ">=17"],
+    "comment": "OpenSSL 3.0"
```

Comment:
FWIW the tests appear to pass with `--openssl-legacy-provider`, see https://ci.nodejs.org/job/citgm-smoker-nobuild/1111/ which uses `NODE_OPTIONS` which we could set via `envVar`.

Reply:
ah I guess that wouldn't work for older versions of Node.js 😞.